lhotse/dataset/diarization.py

```diff
@@ -61,7 +61,7 @@ def __init__(
     def __getitem__(self, cut_ids: Iterable[str]) -> Dict[str, torch.Tensor]:
         cuts = self.cuts.subset(cut_ids=cut_ids)
         return {
-            'features': collate_features(cuts),
+            'features': collate_features(cuts)[0],
```

Comment:
I think this dataset could possibly benefit from adding `features_lens` as a field in batch

Reply:
I am thinking of adding features lens to most of the datasets, maybe except the K2 dataset. WYT?